pkg/multitype/boolstring.go

```diff
@@ -116,3 +117,21 @@ func (boolstr *BoolOrString) Fuzz(c fuzz.Continue) {
 		c.Fuzz(&boolstr.StrVal)
 	}
 }
+
+// BoolOrDefaultFalse returns bool val, if strValu is parsed returns parsed value  else false as default when parse error
+func (boolstr *BoolOrString) BoolOrDefaultFalse() bool {
+	val, _ := boolstr.Bool()
+	return val
+}
```

Comment:
```suggestion
func (boolstr *BoolOrString) BoolOrDefaultFalse() bool {
	val, err := boolstr.Bool()
	if err != nil {
		return false
	}
	return val
}
```
you shouldn't rely on the behavior of something returning non-nil error